diff --git a/slatec/CMakeLists.txt b/slatec/CMakeLists.txt
index d1f215c..a62a394 100644
--- a/slatec/CMakeLists.txt
+++ b/slatec/CMakeLists.txt
@@ -44,6 +44,12 @@ set(MAPHYS_SLATEC
 # -------------------------------
 add_library(slatec ${MAPHYS_SLATEC})
 
+if(MAPHYS_BLASMT AND LAPACK_PAR_LIBRARIES)
+  target_link_libraries(slatec ${LAPACK_PAR_LIBRARIES})
+else()
+  target_link_libraries(slatec ${LAPACK_LIBRARIES})
+endif()
+
 # installation
 # ------------
 install(TARGETS slatec
